apps/aehttpserver/web/controllers/tx_controller.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Aehttpserver.TxController do
+  use Aehttpserver.Web, :controller
+  alias Aecore.Txs.Pool.Worker, as: Pool
+  alias Aecore.Utils.Serialization, as: Serialization
+  alias Aecore.Chain.Worker, as: Chain
+  alias Aecore.Utils.Blockchain.BlockValidation
+
+  def show(conn, params) do
+    account_bin =
+      params["account"]
+      |> Base.decode16!()
+    user_txs = Pool.get_txs_for_address(account_bin, :no_hash)
+    case user_txs do
+      [] -> json(conn, [])
+      _ ->
+        case params["include_proof"]  do
+          "true" ->
+            user_txs_and_hash = Pool.get_txs_for_address(account_bin, :add_hash)
+            merkle_tree = build_tx_tree(user_txs)
+            {size, {key, value, _}} = merkle_tree
+            proof = :gb_merkle_trees.merkle_proof(key, merkle_tree)
```

Comment:
What's the idea behind this "proof" value? It's calculated based on the transactions we send to the user. An useful proof would be if for every transaction we included a Merkle Proof that proves the transaction was included in a particular block, given the user that verifies the proof has the header for that block.

Reply:
yes, this should be done as @cytadela8 said. The user will have the possibility to download the block header by hash and verify this proof